src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -51,15 +51,58 @@ public boolean connect(String database, String user, String password) {
 
     private int getTableRowsCount(String tableName) throws SQLException {
         try (Statement statement = connection.createStatement();
-             ResultSet resultSet = statement.executeQuery(TABLE_ROWS_COUNT_QUERY.replace("?",tableName))) {
+             ResultSet resultSet = statement.executeQuery(TABLE_ROWS_COUNT_QUERY.replace("?", tableName))) {
             resultSet.next();
             return resultSet.getInt(1);
         }
     }
 
-    public void closeConnection() {
+    public DataSet[] getTableData(String tableName) {
+        String query = String.format("SELECT * FROM \"%s\"", tableName);
+        DataSet[] tableData = new DataSet[0];
+        if (connection == null) {
+            return tableData;
```

Comment:
See that materials about one/many return from methods in Java
https://stackoverflow.com/questions/15527482/is-using-return-at-the-start-of-method-bad-coding-practice
https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement
and this one
https://dzone.com/articles/common-code-violations-java

Reply:
I tried to change the logic. @ivshapovalov  please look it.